Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -195,7 +201,9 @@ private void CreateRange()
             else
             {
                 this.lower = limits[0];
-                this.upper = limits[1];
+                this.upper = limits[1] > Int32.MaxValue
+                    ? limits[1]
+                    : limits[1] + 1;

```

Comment:
I'm not sure I follow this either. How does including `Int32.MaxValue` prevent the 255th number from being 255?